.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Resolve https://github.com/nimblehq/git-template/issues/??
```

Comment:
I recommend changing these directly, so you wont' have to do it later 😇 (same for all the templates :))

You can set it like the way you used in this PR 🚀 

```
- close #
```